pkg/apis/network/v1/pki_types.go

```diff
@@ -0,0 +1,60 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
The documentation in this file completely fails to document the type...

Reply:
so it does.